lib/dynflow/persistence.rb

```diff
@@ -93,6 +93,12 @@ def save_delayed_plan(delayed_plan)
       adapter.save_delayed_plan(delayed_plan.execution_plan_uuid, delayed_plan.to_hash)
     end
 
+    def freeze_delayed_plan(execution_plan_id, frozen = true)
+      plan = load_delayed_plan(execution_plan_id)
+      plan.frozen = frozen
+      save_delayed_plan(plan)
+    end
```

Comment:
I can split this into a freeze and unfreeze method if preferred.

Reply:
either split or `set_freeze`, as it would indicate more that it's about setting the `frozen` flag, not just setting it to true (which it reads like right now)